.changeset/early-eyes-bow.md

```diff
@@ -0,0 +1,6 @@
+---
+'astro': minor
+'@astrojs/markdoc': patch
```

Comment:
Should this be a minor as the new version won't work in older versions of Astro anymore? (Due to content flag change)

Reply:
Agreed, will bump